src/SFA.DAS.EmployerIncentives.Functions.AcceptanceTests/SFA.DAS.EmployerIncentives.Functions.AcceptanceTests.csproj

```diff
@@ -31,50 +31,4 @@
     <ProjectReference Include="..\SFA.DAS.EmployerIncentives.Infrastructure\SFA.DAS.EmployerIncentives.Infrastructure.csproj" />
   </ItemGroup>
 
-  <ItemGroup>
```

Comment:
Is it ok to remove these?

Reply:
that's a really good question, looking at it again. These mark the cs files as 'to be generated', will correct this thx

Reply:
Looking at this again and I really don't think we need to keep these directives in the csproj file. It all appears to work perfectly well without these sections and files are regenerated automatically within the MSBuild process. 

Let me know your thoughts on this Dave

Reply:
Ok, just wanted to check 👍